draft-terbu-sd-jwt-vc.md

```diff
@@ -255,6 +253,12 @@ exist between `sub` and `cnf` claims.
 Additional public claims MAY be used in SD-JWT VCs depending on the
 application.
 
+#### SD-JWT VC without selectively disclosable claims
+
+SD-JWT VC may not contain any selectively disclosable claims.
+In that case, the SD-JWT VC MUST NOT contain the `_sd` claim in the JWT body. I also
+MUST NOT have any Disclosures.
```

Comment:
```suggestion
MUST NOT have any Disclosures. For issuance, the trailing `~` MUST BE omitted from the SD-JWT.
For presentation the trailing `~` MUST BE omitted unless a Key Binding JWT is present.
```

Reply:
if we keep approach in this PR, `~` needs to stay even when there is no SD.